tests/pos/i22193.scala

```diff
@@ -0,0 +1,78 @@
+
+def fn2(arg: String, arg2: String)(f: String => Unit): Unit = f(arg)
+
+def fn3(arg: String, arg2: String)(f: => Unit): Unit = f
+
+def test() =
+
+  fn2(arg = "blue sleeps faster than tuesday", arg2 = "the quick brown fox jumped over the lazy dog"): env =>
+    val x = env
+    println(x)
+
+  // doesn't compile
```

Comment:
hopefully, not true anymore :)

Reply:
I used to add `// was:` to mean previously, which didn't help, but then I noticed Odersky just leaves the original text. I like it, it's like the test has a cherished patina.